src/components/inputs/Select.md

```diff
@@ -45,23 +47,27 @@ const items = [
 		)
 	}
 ];
-const [selected, setSelected] = useState(4);
+const [selected, setSelected] = useState();
+const onChange = (ev) => {
+    console.log(ev);
+    setSelected(ev);
+};
 <>
 	<Select
 		items={items}
 		background="gray5"
 		label="Select an item"
-		onChange={setSelected}
-		defaultSelection={{ value: '4', label: 'goodnight' }}
+		onChange={onChange}
 	/>
-	<Text>Currently selected: {items[selected - 1].label}</Text>
+    <Text>Currently selected: {selected ? items[selected - 1].label : 'no selection'}</Text>
```

Comment:
Selected is a string. There is a type coercion here that is dangerous imo (there was also before in other examples, but it might be right moment to fix it since we are refactoring)